src/roseingrave/_sheets.py

```diff
@@ -9,6 +9,7 @@
 # ======================================================================
 
 import google.auth.exceptions
+import gspreed.client
```

Comment:
Could you replace these import statements with `import gspread` instead? I believe that should cover everything.